### ISSUE TITLE
[bug]  some button text appears black on night mode

### ISSUE THREAD

**Issue by personalcodecare** [reown-com/appkit#3871]
### Link to minimal reproducible example

https://www.flipbase.io/

### Summary

In night mode, some button text appears black, making it difficult to read. This affects usability and accessibility. Please review and adjust the text color for better contrast.

### List of related npm package versions

@reown/appkit: "^1.6.4",
@reown/appkit-adapter-wagmi: "^1.6.7",
wagmi: "^2.14.11",

### Node.js Version

v22.11.0

### Package Manager

10.9.0

**Comment by Sam-Newman** [comment]
Thanks for raising this -  did you try and adjusting the accent for the buttons?<br><br>you can try out different configurations at [https://demo.reown.com/](https://demo.reown.com/)

**Comment by personalcodecare** [comment]
LGTM on Demo.
Thanks!
I'll test more.
